doc/requirements-doc.txt

```diff
@@ -11,6 +11,7 @@ opencv-python-headless
 pandas
 pickle5
 pillow
+pyarrow
```

Comment:
Can you add `pyarrow` in `python/setup.py` and `ci/travis/install-dependencies.sh`. And I think we shopuld enforece pyarrow version to ge greater than 0.15

Reply:
Thanks! This is definitely not the right location.